README.markdown

```diff
@@ -78,6 +78,7 @@ Next, add the following to your application.rb file, or before you perform a coo
 
     Timezone::Configure.begin do |c|
       c.google_api_key = 'your_google_api_key_goes_here'
+      c.google_client = 'your_google_client_id' # only if using 'Google for Work'
```

Comment:
Using `google_client_id` would be preferable since it more clearly matches the documentation and helps indicate that it's an identifier instead of just an object.